### PR TITLE
operators tackle-operator (2.0.0-beta.0)

### PR DESCRIPTION
**Pull request by fbladilo**
Thanks submitting your Operator. Please check below list before you create your Pull Request.

### New Submissions

* [x] Are you familiar with our [contribution guidelines](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-via-pr.md)?
* [ ] Have you [packaged and deployed](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md) your Operator for Operator Framework?
* [ ] Have you tested your Operator with all Custom Resource Definitions?
* [ ] Have you tested your Operator in all supported [installation modes](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#operator-metadata)?
* [ ] Have you considered whether you want use [semantic versioning order](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md#semver-mode)?
* [x] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?
* [x] Is operator [icon](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#operator-icon) set?

### Updates to existing Operators

* [ ] Did you create a `ci.yaml` file according to the [update instructions](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md)?
* [ ] Is your new CSV pointing to the previous version with the `replaces` property if you chose `replaces-mode` via the `updateGraph` property in `ci.yaml`?
* [ ] Is your new CSV referenced in the [appropriate channel](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#channels) defined in the `package.yaml` or `annotations.yaml` ?
* [ ] Have you tested an update to your Operator when deployed via OLM?
* [x] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?

### Your submission should not

* [ ] Modify more than one operator
* [ ] Modify an Operator you don't own
* [ ] Rename an operator - please remove and add with a different name instead
* [ ] Modify any files outside the above mentioned folders
* [ ] Contain more than one commit. **Please squash your commits.**

### Operator Description must contain (in order)

1. [ ] Description about the managed Application and where to find more information
2. [ ] Features and capabilities of your Operator and how to use it
3. [ ] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [ ] Human readable name and 1-liner description about your Operator
* [ ] Valid [category name](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#categories)<sup>1</sup>
* [ ] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [ ] Links to the maintainer, source code and documentation
* [ ] Example templates for all Custom Resource Definitions intended to be used
* [ ] A quadratic logo

Remember that you can preview your CSV [here](https://operatorhub.io/preview).

--

<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need

<sup>2</sup> For more information see [here](https://sdk.operatorframework.io/docs/overview/#operator-capability-level)

### CODE REVIEW

File: operators/tackle-operator/2.0.0-beta.0/metadata/annotations.yaml
Comment (mvalarh) [inline operators/tackle-operator/2.0.0-beta.0/metadata/annotations.yaml:15]: Please remove these 2 lines since there is no `tests/scorecard/` in bundle

**Review by mvalarh — CHANGES_REQUESTED** [review]

File: operators/tackle-operator/2.0.0-beta.0/metadata/annotations.yaml
Comment (fbladilo) [reply operators/tackle-operator/2.0.0-beta.0/metadata/annotations.yaml]: Removed

**Review by mvalarh — APPROVED** [review]

**Comment by mvalarh** [comment]
@PhilipCattanach,@carlosthe19916,@m-brophy,@jonathanvila,@mrizzi , please approve

**Comment by mvalarh** [comment]
@fbladilo can you fix conflict?

**Comment by mrizzi** [comment]
> @PhilipCattanach,@carlosthe19916,@m-brophy,@jonathanvila,@mrizzi , please approve

After https://github.com/k8s-operatorhub/community-operators/pull/1231 has been merged, we're expecting this PR to automerge once @fbladilo will merge latest main (which has to be done to fix the conflict).
In this way it will be proved the added owners works as expected with no need for manual approval, ensuring the new owners have full autonomy in PRs.

**Pull request merged**
